fastlane/lib/fastlane/actions/nexus_upload.rb

```diff
@@ -107,7 +128,7 @@ def self.available_options
                                        optional: false),
           FastlaneCore::ConfigItem.new(key: :mount_path,
                                        env_name: "FL_NEXUS_MOUNT_PATH",
-                                       description: "Nexus mount path",
+                                       description: "Nexus mount path (note: Nexus 3 instances have this configured as empty by default)",
```

Comment:
Can we remove the `note: ` before the Nexus 3 part?

Reply:
Sure.